setup.py

```diff
@@ -484,6 +484,7 @@ def get_sphinx_theme_version() -> str:
     'click~=7.1',
     'coverage',
     'docutils',
+    'filelock',
```

Comment:
Looks like the py-filelock project is no longer maintained, last commit to the project was in 2019. Apart from that, this LGTM

Reply:
Do you think that matters @ephraimbuddy ? This is rather simple library and I do not expect locking (300 lines of code). Looks like it is not updated because it simply does it job well :)

Reply:
You're right @potiuk :)